internal/proto/codec/list_iterator_codec.go

```diff
@@ -1,3 +1,6 @@
+//go:build !cover
+// +build !cover
```

Comment:
I guess these are for excluding from coverage? If so, why? I guess because they are not used in client.

Reply:
They are for exclusion yes. At the time I didn't want to remove these codecs because I wasn't sure if we will going to use it.
For now we decided that we will remove these, and address the actual problem in @yuce's ongoing work on client-protocol. I removed this and the ones that are similar to this at https://github.com/hazelcast/hazelcast-go-client/pull/703/commits/48594298688387b76ed9fd3280296350ca7355fb